src/App.vue

```diff
@@ -510,17 +521,21 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
    * NOTE: This is only implemented for Alteration filings atm.
    * @param action the emitted action
    */
-  private async handleFeeSummaryActions (action: SummaryActions): Promise<void> {
+  private async handleFeeSummaryActions (action: FeeSummaryActions): Promise<void> {
     switch (action) {
-      case SummaryActions.SAVE_RESUME_LATER:
+      case FeeSummaryActions.BACK:
+        this.setSummaryMode(false)
+        await this.scrollToTop(document.getElementById('app'))
+        break
+      case FeeSummaryActions.SAVE_RESUME_LATER:
         // Save filing and return to dashboard.
         await this.onClickSave()
-        this.goToDashboard()
+        this.goToDashboard(true)
```

Comment:
This was previously not "forced" in case save failed.

Reply:
`onClickSave()` may clear the Have Changes flag on successful save, so you should not need to force here.

Reply:
Ok, will look into this!